### DISCUSSION TITLE
Pro-HD choppy tuner video on some channels

### DISCUSSION THREAD

**Discussion by mike-uk** [GerryDazoo/Slinger#571] [Q&A] [answered]
Thank you Gerry for making this software and every one replied to my message. My Pro-HD box is now working!

I just have a little question, regarding video quality, on some tuner channels it's a bit choppy but some are fine. I tried to turn down the resolution in config.ini but it doesn't seem to change anything as the picture look the same (I do see the setting updated in the log). I am thinking if it is my orange pi zero LTS (H2+ A7 1.2Ghz Quadcore 512mb ram) being too slow (it is connected by 100mbps ethernet not wifi) or my Slingbox is connected to a powerline network adaptor. The antenna is from the roof and the tv works fine with it, or would I need to do a full detail scan?

**Comment by BobC1954** [comment, marked as answer]
Mike. I have exactly the same issue on certain channels. I'm 99% sure that Arqiva have done something in their transmitter stream (signal) that is causing this issue. It started about 10 weeks (or so) ago. Checking signal strength from the antenna with a TV in place of the Slingbox shows 100% signal strength and 100% signal quality - and no choppiness. So I'm pretty sure the transmitted signal has a compatible issue with Slingbox. Edit. Arqiva themselves use (or used to use) Slingboxes to monitor their remote transmitter sites and I spent quite a lot of time working with them to identify the tuning crash issue some years ago, that resulted in Sling issuing the .420 firmware for UK. This was also because of changes they made to the transmitted stream. Unfortunately, we no longer have a "Sling" company who could make firmware changes - if necessary, so I think we have to live with it. The problem (for me at least) happens with Gerry's software or with official Slingplayers.

**Reply by mike-uk** [reply]
@BobC1954 I checked through the list it seems the channels affected are the ones belonging to ITV, Channel 4 and Channel 5. All their sub-channels are the same. I have ordered a cheap DVB-T2 box with YPbPr out from Aliexpress hopefully can fix the problem. Its funny that all the TVs still works but slingbox need an upgrade. Thank you for answering my question!

**Reply by BobC1954** [reply]
Yes, exactly the same as me. I'm having a dig round to see if I can find my contact in Arqiva. Not sure if he still works there.

**Reply by BobC1954** [reply]
Mike. Which transmitter site in UK do you receive signals from?

**Reply by mike-uk** [reply]
@BobC1954 Its Sutton Coldfield in West Midlands

**Reply by BobC1954** [reply]
Ah that's interesting because I use the Wrekin. It gets it signal from Sutton Coldfield. I'm just digging through my emails from 2011 to find my contact. Then see what I can find out. Will let you know.

**Reply by mike-uk** [reply]
@BobC1954 Thanks Bob, thats interesting, I didn't know they relay signal like that.

**Reply by mike-uk** [reply]
@BobC1954 It is quite funny that I enjoy watching tv from slingbox than live tv. The picture just look different, it feel like what tv used to be in the 90s early 00s have optical feel to it. HD tv is too clear even at the background its like watching outside a windows in real life which hard to pay attention to.
